### DISCUSSION TITLE
Disable matching bracket highlighting on very close pairs

### DISCUSSION THREAD

**Discussion by klotzambein** [helix-editor/helix#8239] [General]
## Issue
When using the theme gruvbox I noticed that it can be confusing when the cursor is on a bracket with a pair right next to it, see example below. This issue is less relevant in different themes, but still can sometimes be confusing. 

## Example
```text
fn foo() {}
      ^^ ^^
```

## Feature Request
My suggestion is to disable bracket highlighting in cases highlighted above. This should have no effect on all other bracket matching functionalities.

## Draw backs
- Unneeded complexity: this feature has the potential to confuse newcomers.
- Nested brackets: in certain cases the current matching might be advantageous: `((()(()))())`.

**Comment by kirawi** [comment]
Maybe as a plugin.
